changelog.md

```diff
@@ -10,6 +10,7 @@
 - [#4001](https://github.com/ignite/cli/pull/4001) Improve `xgenny` dry run
 - [#3967](https://github.com/ignite/cli/issues/3967) Add HD wallet parameters `address index` and `account number` to the chain account config
 - [#4004](https://github.com/ignite/cli/pull/4004) Remove all import placeholders using the `xast` pkg
+- [#4076](https://github.com/ignite/cli/pull/4076) Remove ts relayer commands
```

Comment:
nit: can we list exactly which commands are removed? (ignite relayer behavior has changed and ignite tools is gone)

Reply:
```suggestion
- [#4076](https://github.com/ignite/cli/pull/4076) Remove the ignite `realyer` and `tools` commands with all ts-realyer logic
```